src/main/java/com/conveyal/r5/analyst/cluster/AnalystWorker.java

```diff
@@ -327,84 +326,65 @@ private void handleOneRequest(GenericClusterRequest clusterRequest) {
                 e.printStackTrace();
             }
             if (random.nextInt(100) >= dryRunFailureRate) {
-                deleteRequest(clusterRequest);
+                deleteRequest(request);
             } else {
-                LOG.info("Intentionally failing to complete task for testing purposes {}", clusterRequest.taskId);
+                LOG.info("Intentionally failing to complete task for testing purposes {}", request.taskId);
```

Comment:
Seeing "request.taskId" reinforces my thought that we should just call them "tasks". `AnalysisTask task; task.taskId` makes sense to me.